### ISSUE TITLE
Nether Protection

### ISSUE THREAD

**Issue by kaytotes** [Project-Epoch/bugtracker#2870]
**Spell ID's:** 30302

> should trigger on ticks from hellfire

**Comment by ghost** [comment]
Needs better confirmation, Hellfire did not proc Nether Protection in Burning Crusade Classic from my own findings and if it is supposed to proc this opens a can of worms. Hellfire is a positive spell, despite it dealing damage. That would mean Nether Protection can proc from positive spells, such as applying Demon Armor to yourself, unless this is specifically scripted in some way.

It could proc from Hellfire when Burning Crusade launched, but it looks like this was removed at some point during the expansion. However, it is difficult to confirm that.

It looks like its only supposed to proc from the initial hit of negative spells. Meaning another warlock casting Fear on you could proc it, and the initial cast of Corruption on you could proc it, but the ticks of Corruption could not.

**Comment by Bennylavaa** [comment]
Doing some digging it looks like TBC classic had some bugs with hellfire where it didnt work as it should. 
https://us.forums.blizzard.com/en/wow/t/nether-protection-not-procing-from-hellfire/985342

Also, here is some bug report for some other server on this: https://github.com/SunstriderEmu/BugTracker/issues/1386 

From wowhead https://www.wowhead.com/tbc/spell=30299/nether-protection#comments 
<img width="1219" alt="Screenshot 2024-09-19 at 5 42 40 PM" src="https://github.com/user-attachments/assets/759567ea-5b0b-4205-aefe-27a489f817fd">

Here has a list of affected spells as well: https://www.wowhead.com/tbc/spell=30301/nether-protection 
<img width="711" alt="Screenshot 2024-09-19 at 5 45 17 PM" src="https://github.com/user-attachments/assets/49a45795-ab9d-4e90-a9d8-ba5290efd604">

Although this isnt proof other tbc private servers it procs on hellfire as well such as stormforge. 

With the above info it sounds like it worked in 2.4.3 but in classic it didnt work after some time so I guess it is up to you guys however I can say I will miss this.

**Comment by Shadownflame** [comment]
Yep this is a typical case where blizzard is not blizzlike. I'd suggest epoch goes with the original 2007 implementation because the synergy between these effects was quite important for fire locks and fire locks are far from an overpowered spec here.
